README.md

```diff
@@ -19,6 +19,18 @@ The DaemonSet yaml file has two environment variables. Fluentd uses these variab
 |   LOGZIO_TOKEN       | [Logz.io account token](https://app.logz.io/#/dashboard/settings/general) |
 |   LOGZIO_URL         | Logz.io listener url, If the account is in the EU region insert https://listener-eu.logz.io:8071. Otherwise, use https://listener.logz.io:8071. You can tell your account's region by checking your login URL - app.logz.io means you are in the US. app-eu.logz.io means you are in the EU |
 
+For improved security you should store the variables using Kubernetes secrets:
+
+```
+kubectl create secret generic logzio-secrets --from-literal=logzio_token='MY_LOGZIO_TOKEN' --from-literal=logzio_url='MY_LOGZIO_URL' -n kube-system
```

Comment:
@idohalevi I have a draft for k8s metrics that's not quite out the door. Thinking ahead here—should we change variable names here to make sure secrets for logs and metrics are discretely named?

That way, users can have both log and metrics daemonsets installed—with their secrets—in k8s without issue. That should also keep users from running into naming conflicts if they don't modify our default config files.

Reply:
@idohalevi Also, same note on calling the secret `logzio-secrets`. Maybe renaming to `logzio-logs-secret` or something like that.